hypothesis-python/src/hypothesis/control.py

```diff
@@ -171,7 +173,7 @@ def target(observation: float, *, label: str = "") -> None:
     check_type(str, label, "label")
 
     context = _current_build_context.value
-    if context is None:
+    if context is None and allow_outside_given_test is False:
```

Comment:
```suggestion
    if context is None:
    	if allow_outside_given_test:
    		return
```

And we'll want to `check_type(bool, allow_outside_given_test, "allow_outside_given_test")` above.